apps/website/.vuepress/blocks.js

```diff
@@ -0,0 +1,93 @@
+const md = require('markdown-it')();
+
+module.exports = [
+  // SUMMARY BLOCK
+  [
+    'vuepress-plugin-container',
+    {
+      type: 'summary',
+      render: function (tokens, idx) {
+        const item = tokens[idx];
+        if (item.type === 'container_summary_open') {
```

Comment:
NIT: we could do it with switch for less noise from the `if-else-if`

```ts
switch(item.type) {
  case 'container_summary_open': return `<ClrSummary>`;
  case 'inline': return item.content
  case 'container_summary_close': return `</ClrSummary>`
}

switch(item.type) {
  case 'container_do_open': 
    return `<DocDo summary="${md.renderInline(item.info.replace('do ', ''))}">`;
  case 'html_block':
    return item.content
  case 'container_do_close':
    return `</DocDo>`
}

switch(item.type) {
  case 'container_dont_open':
    return `<DocDont summary="${md.renderInline(item.info.replace('dont ', ''))}">`;
  case 'html_block':
    return item.content
  case 'container_dont_close':
    return `</DocDont>`
}

switch(item.type) {
  case 'container_col_open':
    return `<DocCol summary="${md.renderInline(item.info.trim().substr(3).trim())}">`
  case 'html_block':
    return item.content
  case 'container_col_close':
    return `</DocCol>`
}

switch(item.type) {
  case 'container_inset_open':
    return `<DocInset align="${item.info.replace('inset ', '')}">`;
  case 'html_block': return item.content;
  case 'constainer_inset_close': return `</DocInset>`;
}
```

Reply:
Thanks for the suggestion. I'll add a TODO comment with a link to this comment so we can revisit after launch.